ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -46,10 +50,19 @@ touch ${this.configFilePath || '<path>'}/pki-worker.hcl`;
     }"`;
 
     const tagsText = `tags {
-    type = [${
-      this.workerTags ? this.convertCommaSeparatedValuesToArray(this.workerTags) : '"<tag1>", "<tag2>"'
-    }]
+    ${
+      this.workerTags.length
+        ? this.workerTags
+            .map((tag) => {
+              return `${tag.key} = [${this.convertCommaSeparatedValuesToArray(
+                tag.value
+              )}]`;
+            })
+            .join('\n    ')
+        : 'key = ["<tag1>", "<tag2>"]'
```

Comment:
There's quite a bit going on here for being directly embedded in a string.  Can this be separated out somehow?

Reply:
I extracted it out to a function, let me know if that reads better